conduct/COC_TEAM_CHARTER.md

```diff
@@ -0,0 +1,92 @@
+This document defines how the OpenJS Foundation implements its code of conduct.
+
+If you want to report an incident, go to [code-of-conduct.openjsf.org][CoC].
+
+***
+
+# Code of Conduct Team Charter
+
+The Cross Project Council (CPC) delegates enforcement of the [Code of Conduct][CoC] to the Code of Conduct Team (CoC Team) through this Code of Conduct Team Charter. This charter defines the composition and roles of the CoC Team, along with CoC team responsibilities and operations.
+
+## Responsibilities
+
+[TODO: add links]
+
+The CoC Team's responsibilities are to:
+
+- Handle code of conduct [incident reports][incident report].
+- Handle code of conduct [escalation requests][escalation].
+- Handle code of conduct [appeals][appeal].
+- Keep the CPC up to date of cases requiring the CPC's attention.
+- Provide regular transparency reports to the CPC.
+- Define, document, and maintain all processes necessary to its operations, notably the [CoC Process for Incident Management](COC_PROCESS_FOR_INCIDENT_MANAGEMENT.md).
+- Work with OpenJS Foundation staff to develop infrastructure and processes necessary to comply to the requirements of the [code of conduct policy][CoC Policy], notably when it comes to record keeping, confidentiality, and transparency.
+- Uphold this charter, the [code of conduct][CoC], and the [code of conduct policy][CoC Policy].
+
+## CoC Team Structure
+
+The CoC Team is composed of five (5) primary members and a variable number of alternate community members.
+
+### Primary members
```

Comment:
Alternate membership is never defined. Based on how this team charter is written, this needs to be well-defined. I am blocking on this.

Reply:
Both primary and alternate members are defined in the charter, in terms of their roles. Could you clarify how else you'd want them defined?

Reply:
Please link where both primary and alternate members are defined? This file is called "COC_TEAM_CHARTER.md", so I assume it's the charter you're talking about. Searching throughout the full PR contents, I'm not able to find definitions - I'm bad at Control + F sometimes, though!

The section I left this comment on is titled "Primary members" but there is no "Alternate members" section - looking for the definition of "Alternate member" I realized that Primary members actually _also_ aren't defined here, which it probably should be.

Reply:
Fixed.

Reply:
@bnb can you please check this fix and resolve this conversation if you're happy with the fix?